src/site/stages/build/process-cms-exports/schemas/output/node-story_listing.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
     changed: { type: 'number' },
     entityPublished: { type: 'boolean' },
     entityMetatags: { $ref: 'MetaTags' },
+    entityUrl: { $ref: 'EntityUrl' },
```

Comment:
`entityUrl` should be injected in every transformer's output. I don't think it is necessary to add it here. Can you please verify this?

Reply:
Nope, this wasn't necessary. Removed.